### PR TITLE
New issue templates

### PR DESCRIPTION
**Pull request by aaronliu0130**
### 1. Does this fix an issue in the CatBlock issue tracker? If so, replace the 0 with its issue number below:
No.

### 2. How can we test this? Step by step instructions please.

1. Open a new issue.
2. Watch the auto bug labels and issues.

### 3. What happened before?
Just plain old issue templates.

### 4. What should happen with this fix?
Seriously, it's just plain old issue templates.

### 5. Anything else we should know about this patch?
IT'S JUST PLAIN OLD ISSUE TEMPLATES!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -23,3 +34,5 @@
 
 #### 5. Any other information you think might be useful for us to know?
 
+
+#### 6. Please tell us your browser version.
```
Comment (itskdog) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Browser version is in the debug info, which we already request, as a part of the browser's User Agent string.

**Review by itskdog — COMMENTED** [review]
Hi there @aaronliu0130!

Apologies for the very late review of these changes! All seems okay, though I'm interested in the additions at the top of the issue template. Could you explain what that does? Would that add an option to a list, or as there's only one, does it just use it alone for now?

Thanks very much!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (aaronliu0130) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Okay. That's fixed.

**Comment by aaronliu0130** [comment]
>  Could you explain what that does? Would that add an option to a list, or as there's only one, does it just use it alone for now?

So, let me explain.
"name" and "about" are shown when you create a new issue, github will ask you which issue template you want to use if there are multiple issue templates.
"title" is the default recommended title, however the value is currently blank, which means it won't reccomend any titles.
"lables" is the default label the bug report is assigned."asignees" is the default assignee for the issue, similar to the "lables" tag, however it is blank so it won't assign anyone.

**Comment by aaronliu0130** [comment]
@itskieranp Review required.

**Review by itskdog — APPROVED** [review]
Sorry for the long wait, and thanks for the ping. Looks good, I'll approve.

**Pull request merged**